packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -0,0 +1,153 @@
+                                : PlatformDetails[platform as keyof typeof PlatformDetails].icon}
+                        </div>
+                        <div className='flex flex-col'>
+                            <div className='flex items-center flex-row gap-300'>
+                                <Text size='md'>
+                                    {platform === mt5Platform
+                                        ? MarketTypeDetails[(marketType ?? 'all') as keyof typeof MarketTypeDetails]
+                                              .title
+                                        : PlatformDetails[platform as keyof typeof PlatformDetails].title}{' '}
+                                    {!activeAccount?.is_virtual && details?.landing_company_short?.toUpperCase()}
+                                </Text>
+                            </div>
+                            <Text className='text-system-light-less-prominent-text' size='sm'>
+                                {loginId}
+                            </Text>
+                        </div>
+                    </div>
+                    <div className='flex flex-col items-end'>
+                        <Text bold>{details?.display_balance}</Text>
+                    </div>
+                </div>
+                <div className='flex flex-col gap-100'>
+                    {platform === mt5Platform && (
+                        <Fragment>
+                            <MT5TradeDetailsItem label='Broker' value='Deriv Holdings (Guernsey) Ltd' />
+                            <MT5TradeDetailsItem
+                                label='Server'
+                                value={(details as THooks.MT5AccountsList)?.server_info?.environment ?? 'Deriv-Server'}
+                            />
+                            <MT5TradeDetailsItem label='Login ID' value={loginId ?? '12345678'} />
+                            <MT5TradeDetailsItem label='Password' value='********' variant='password' />
+                        </Fragment>
+                    )}
+                    {platform === dxtradePlatform && (
+                        <Fragment>
+                            <MT5TradeDetailsItem
+                                label='Username'
+                                value={(details as THooks.DxtradeAccountsList)?.login ?? '12345678'}
+                            />
+                            <MT5TradeDetailsItem label='Password' value='********' variant='password' />
+                        </Fragment>
+                    )}
+                    {platform === ctraderPlatform && (
+                        <MT5TradeDetailsItem
+                            value=' Use your Deriv account email and password to login into the cTrader platform.'
+                            variant='info'
+                        />
+                    )}
+                </div>
+                <div className='grid grid-cols-[2.5rem_auto]'>
```

Comment:
Is there a different way to write this it can be confusing for some of us 😅

Reply:
@aizad-deriv Honestly, I didn't find any solution how to do this another way. I'll be glad if you can suggest anything except create constant in tailwind.config.js

Reply:
I'm not sure maybe you can give me a screenshot of the layout then I can help you 😅

Reply:
@aizad-deriv I need to convert `grid-template-columns: 2.5rem auto` to `tailwind`

Reply:
I added at the bottom